assets/shaders/custom_material_import.wgsl

```diff
@@ -0,0 +1,2 @@
+// this is made available to the importing module
+const COLOR_MULTIPLIER: vec4<f32> = vec4<f32>(1.0, 1.0, 1.0, 0.5);
```

Comment:
Missing newline. Maybe configure your editor to ensure there is a newline at the end of files? :)

Reply:
done and done.